packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -231,12 +231,26 @@ class PerformanceModeRequestHandle {
 ///   priority and are executed in priority order according to a
 ///   [schedulingStrategy].
 mixin SchedulerBinding on BindingBase {
+  /// Override [_enableProfileFrame] in debug mode.
+  @visibleForTesting
+  static bool? debugOverrideEnableProfileFrame;
```

Comment:
I don't think we should be exposing this and its also not super-helpful for testing. The test now only checks that this flag works which only exists for the test. It doesn't really test whether the actual env variable is doing the right thing...

Reply:
So may I know how Flutter tests existing env variable? Then I can make something similar

Reply:
I can remove the environment variable, see below.